runtime/tests/accounts.rs

```diff
@@ -128,11 +130,29 @@ fn test_bad_bank_hash() {
         db.store_cached((some_slot, &account_refs[..]), None);
         for pass in 0..2 {
             for (key, account) in &account_refs {
-                assert_eq!(
-                    db.load_account_hash(&ancestors, key, Some(some_slot), LoadHint::Unspecified)
+                if pass == 1 {
+                    assert_eq!(
+                        db.load_account_hash(
+                            &ancestors,
+                            key,
+                            Some(some_slot),
+                            LoadHint::Unspecified
+                        )
                         .unwrap(),
-                    AccountsDb::hash_account(*account, key)
-                );
+                        AccountHash(Hash::default())
+                    );
+                } else {
+                    assert_eq!(
+                        db.load_account_hash(
```

Comment:
should we ever be saving/loading a hash now? not sure about this test.

Reply:
I think the intention of this test is to test load hash from cache vs storage. 
What this test does is to make sure that when we load from the write cache, we should have the correct hash (pass=0), but when we load from the storage, we should have default hash (pass=1).